azure-monitor-opentelemetry-distro/azure/monitor/opentelemetry/distro/distro.py

```diff
@@ -34,6 +37,7 @@ def _configure(self, **kwargs) -> None:
 
 def _configure_auto_instrumentation() -> None:
     try:
+        log_status(False, "Distro being configured.")
```

Comment:
Seems kinda strange that diagnostic logging has it's own class but status logger doesn't. Should we do something similar like `AzureStatusLogger.log(...)`?

Reply:
We can do this. But it would really just be cosmetic. There's no class or instance variables needed.